internal/resource/common/defaultref/default_ref_schema.go

```diff
@@ -0,0 +1,15 @@
+package defaultref
+
+import (
+	"github.com/hashicorp/terraform-plugin-framework/provider/schema"
+	"github.com/pingidentity/terraform-provider-pingfederate/internal/resource/common/resourcelink"
+)
+
+func ToSchema(description string) schema.Attribute {
```

Comment:
IMO we don't really need this as a separate package if it's just directly calling the other method.

Reply:
removed